src/components/RepositoryDetail/RepositoryDetail.module.scss

```
@@ -0,0 +1,12 @@
+.metadata {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    grid-column-gap: 2rem;
```

Stylelint (17.10.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)